tests/common/test_cgroupapi.py

```diff
@@ -303,7 +303,8 @@ def test_it_should_return_extension_slice_name(self):
         self.assertEqual(extension_slice_name, "system-walinuxagent.extensions-Microsoft.Azure.DummyExtension_1.0.slice")
 
     @skip_if_predicate_false(i_am_root, "Test does not run when normal user")
-    def test_if_extensions_root_slice_is_created(self):
+    @attr(permissions='root')
```

Comment:
suggestion: I'd made this a bool instead (e.g. @attr(requires_root_privileges)); that makes test selection a little easier

Reply:
Sure, sounds good.